basicMysql/mysqlClass.php

```diff
@@ -0,0 +1,280 @@
+<?php
+  //vendor is folder realted to composer and here used for 'Guzzle'
+  //require("../vendor/autoload.php");
+  use PHPMailer\PHPMailer\PHPMailer;
+
+  //Client is a class which coming from GuzzleHttp and Guzzle used it for email validation.
+  use GuzzleHttp\Client;
+  class MysqlMethods
+  {
+    //Login user-name
+    public $getname;
+    //Login user-password
+    public $getpwd;
+    //Login user-name error message
+    public $status_name = false;
+    //Login user-password error message
+    public $status_pwd = false;
+
+    /**
+    * function getLogin($username, $userpwd)
+    *
+    * @param [string] $username : This contains the user-name.
+    * @param [string] $userpwd : It contains user-password.
+    * This function first check user-name and password from database and then, if data are valid then return true otherwise return false.
+    *
+    * @return boolean
+    */
+    function getLogin($username, $userpwd){
+      $this->getname = $username;
+      $this->getpwd = $userpwd;
+
+      $conn = new mysqli("localhost", 'root', 'Abhi4531@my', 'User_DB');
+      //$sql = "SELECT userName, userPwd FROM User WHERE userName = '$username' AND userPwd = '$userpwd'";
+      $sql = "SELECT userName, userPwd FROM User WHERE userName = '$username'";
+
+      $result = $conn->query($sql);
+
+      if ($result->num_rows > 0) {
+        while($row = $result->fetch_assoc()) {
+          if($userpwd === $row["userPwd"]){
+            return true;
+          }
+        }
+        $this->status_pwd = true;
+        return false;
+      }
+      else {
+        $this->status_name = true;
+        return false;
+      }
+      $conn->close();
+      /*if($username === "Abhi" && $userpwd === "abhi@45"){
+        return true;
+      }else{
+        if($username!="Abhi" && $userpwd!="abhi@45"){
+          $this->status_name=true;
+          $this->status_pwd=true;
+          return false;
+        }
+        else if($username!="Abhi"){
+          $this->status_name=true;
+          return false;
+        }
+        else if($userpwd!="abhi@45"){
+          $this->status_pwd=true;
+          return false;
+        }
+      }*/
+    }
+
+    /**
+     * function getRegister($username, $userpwd, $usermobile, $useremail)
+     * @param [string] $username : It contains user-name
+     * @param [string] $userpwd : It contains user-password
+     * @param [string] $usermobile : It contains user-mobile number
+     * @param [string] $useremail : It contains user-email
+     * This function first get the data from 'FORM' and then check the username and user-password already exist or not,
+     * if exist then it will not store the data into database or if username or user-password does't exist then this
+     * function will store the data into database.
+     * @var boolean
+     */
+    public $unique_status = false;
```

Comment:
We could declare all the class variables before instead of declaring before the function. It is preferred to use a camel case for the class variables.